lib/occupier/postgres/client.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module Occupier
+  module Postgres
+    # @api public
+    class Client
+      extend Forwardable
+
+      attr_reader :environment
+
+      def initialize(environment = 'development', logger = nil)
+        @environment = environment.to_s
+        @logger = logger
+      end
+
+      def create(database_name)
+        raise 'database already exists' if database_exists? database_name
+
+        connect.execute("CREATE DATABASE \"#{database_name}\"")
+      end
+
+      def close
+        ActiveRecord::Base.connection.disconnect!
+      end
+
+      def connect(database_name = nil)
+        connect_to(database_name)
+
+        ActiveRecord::Base.connection
+      rescue StandardError => e
+        raise "Could not connect to database: #{e.message}"
```

Comment:
Could be better to just let it bubble up, we'll be hiding the original stack trace here. Maybe wrap the exception instead?